pkg/controllers/node/node_controller_test.go

```diff
@@ -102,6 +150,16 @@ func TestNodeReconciler_Reconcile(t *testing.T) {
 				},
 			}
 
+			if tt.name == "Node not found" {
```

Comment:
Can we use below to mock SyncNodeStore return nil or error? then I think we do not need to change files like node.go and store.go under nsx/services/node.
patches.ApplyMethod(reflect.TypeOf(r.NodeService), "SyncNodeStore", func(_ *node.Service, nodeName string, deleted bool) error {
					return nil				})

Reply:
ok, fixed